tests/Functional/RoutingTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\MediaBundle\Tests\Functional\Routing;
+
+use Sonata\MediaBundle\Tests\App\AppKernel;
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+/**
+ * @author Javier Spagnoletti <phansys@gmail.com>
+ */
+final class RoutingTest extends WebTestCase
+{
+    /**
+     * @group legacy
+     *
+     * @dataProvider getRoutes
+     */
+    public function testRoutes(string $name, string $path, array $methods): void
```

Comment:
You could also check the routes using and omitting "format" parameter ([ref](https://github.com/sonata-project/SonataUserBundle/blob/cc8ff3fb7645220db7def84e01a7d3066cfd85d9/tests/Functional/Routing/RoutingTest.php#L40-L72)).